packages/platforms/dni/x86-64/x86-64-dni-wb2448/platform-config/r0/src/python/x86_64_dni_wb2448_r0/__init__.py

```diff
@@ -0,0 +1,16 @@
+from onl.platform.base import *
+from onl.platform.dni import *
+
+class OnlPlatform_x86_64_dni_wb2448_r0(OnlPlatformDNI,
+							OnlPlatformPortConfig_48x1_4x10):
+    PLATFORM='x86-64-dni-wb2448-r0'
+    MODEL="wb2448"
+    SYS_OBJECT_ID=".1"
```

Comment:
@jnealtowns Thanks for your reply.
We will follow your suggestion in next commit.